launcher/src/main/resources/launch.properties

```diff
@@ -27,9 +27,9 @@ carbon.initial.osgi.bundles=\
   file\:plugins/org.wso2.carbon.base_5.1.0.SNAPSHOT.jar@2\:true,\
   file\:plugins/org.eclipse.equinox.simpleconfigurator_1.1.0.v20131217-1203.jar@3\:true
 
-osgi.install.area=file\:${profile}
-osgi.configuration.area=file\:${profile}/configuration
-osgi.instance.area=file\:${profile}/workspace
+osgi.install.area=file\:profiles/${profile}
+osgi.configuration.area=file\:profiles/${profile}/configuration
+osgi.instance.area=file\:profiles/${profile}/workspace
```

Comment:
Instead of hard coding this (profiles), can't we get the profile path same as profile name, ${profile}